packages/template-retail-react-app/app/components/product-tile/index.jsx

```diff
@@ -104,26 +125,13 @@ const ProductTile = (props) => {
                 {/* Title */}
                 <Text {...styles.title}>{localizedProductName}</Text>
 
-                {/* Price */}
-                <Text {...styles.price} data-testid="product-tile-price">
-                    {hitType === 'set'
-                        ? intl.formatMessage(
-                              {
-                                  id: 'product_tile.label.starting_at_price',
-                                  defaultMessage: 'Starting at {price}'
-                              },
-                              {
-                                  price: intl.formatNumber(price, {
-                                      style: 'currency',
-                                      currency: currency || activeCurrency
-                                  })
-                              }
-                          )
-                        : intl.formatNumber(price, {
-                              style: 'currency',
-                              currency: currency || activeCurrency
-                          })}
-                </Text>
+                {/*/!* Price *!/*/}
+                {/*Price and discount price will show for first variant for now. We will implement the swatch into PLP later*/}
```

Comment:
Does this mirror what we show on the PDP?

Reply:
It is not currently, but I believe we have a ticket to fix PDP, so I don't do it in this PR